### ISSUE TITLE
[BUG] KP2A keyboard occasionally called up unexpectedly

### ISSUE THREAD

**Issue by WebworkrNet** [PhilippC/keepass2android#2602]
### Checks

- [X] I have read the FAQ section, searched the open issues, and still think this is a new bug.

### Describe the bug you encountered:

Two different scenarios:

1. URL input in the address bar of the browser
2. Text input in a form.

The current keyboard changes unexpectedly to the KP2A keyboard.

### Describe what you expected to happen:

The KP2A keyboard should only be called up for the corresponding input fields or by actively changing the keyboard.

### What version of Keepass2Android are you using?

1.10-pre

### Which version of Android are you on?

Android 14, One UI 6.0

**Comment by WebworkrNet** [comment]
Samsung Galaxy A13 5G (SM-A136B)
no root

**Comment by WebworkrNet** [comment]
No extensions installed

**Comment by skick3** [comment]
Same here since App Version 1.11-r0
As soon as you open the app, you first have to define the K2A keyboard as the default keyboard before you can log in

Device: Pixel 6, Android 14, Systemupdate April 2024

**Comment by PhilippC** [comment]
is the last preference option in settings - app - security enabled? Should say something like "Use integrated keyboard in KP2A"?

**Comment by skick3** [comment]
Yes, that was it. After I deactivated the setting, I was able to open the app with the Gboard keyboard without any problems and messages
